crates/miden-lib/asm/account_components/rpo_falcon_512_procedure_acl.masm

```diff
@@ -22,7 +22,7 @@ const.AUTH_TRIGGER_PROCS_MAP_SLOT=2
 #!
 #! Inputs:  [AUTH_ARGS, pad(12)]
 #! Outputs: [pad(16)]
-export.auth__tx_rpo_falcon512_procedure_acl.2
+export.auth__tx_rpo_falcon512_acl.2
```

Comment:
Nit: I would then also rename the file name to drop "procedure".

Reply:
will be covered by https://github.com/0xMiden/miden-base/pull/1711